index.js

```diff
@@ -750,16 +769,92 @@ alexa.app = function(name) {
     },
     askcli: function(invocationName) {
       var model = skillBuilderSchema();
+      var { prompts, dialog } = skillBuilderDialog();
       model.invocationName = invocationName || self.invocationName || self.name;
       var schema = {
         interactionModel: {
-          languageModel: model
+          languageModel: model,
+          dialog,
+          prompts
         }
       };
       return JSON.stringify(schema, null, 3);
     }
   };
 
+  var skillBuilderDialog = function() {
+    var schema = {
+        dialog: {
+          intents: []
+        },
+        prompts: []
+      },
+      intentName,
+      intent,
+      key;
+
+    var creatPrompt = function(promptId, variations) {
+      return {
+        id: promptId,
+        variations: variations.map(prompt => ({
```

Comment:
I wonder if this should check if the user's passed in an already-constructed response object, and if so just use that instead of string-concatenating SSML for them? 

I'm not sure if I'm being silly, and maybe there's no reason anyone would want to do that in practice, but seeing SSML being manually constructed like this usually gives me pause.